commons-jcs-core/src/main/java/org/apache/commons/jcs3/auxiliary/disk/jdbc/mysql/MySQLDiskCacheFactory.java

```diff
@@ -102,11 +102,8 @@ protected void scheduleOptimizations( final MySQLDiskCacheAttributes attributes,
                 try
                 {
                     final Date[] dates = ScheduleParser.createDatesForSchedule( attributes.getOptimizationSchedule() );
```

Comment:
Would you please consider documenting "never null" as the return value of ScheduleParser.createDatesForSchedule()? The same goes for other calls.